src/tests/tests/background.js

```diff
@@ -357,4 +358,53 @@ QUnit.test("subdomains on the yellowlist are preserved", (assert) => {
   );
 });
 
+QUnit.test("migrates misattributed MDFP domains from snitch map", (assert) => {
```

Comment:
We might want to break this up into three separate tests, especially once you add the extra assertions for action map contents (see below).